x/bank/keeper.go

```diff
@@ -84,7 +103,11 @@ func (keeper BaseKeeper) InputOutputCoins(
 type SendKeeper interface {
 	ViewKeeper
 
+	SetCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) sdk.Error
+	SubtractCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) (sdk.Coins, sdk.Tags, sdk.Error)
+	AddCoins(ctx sdk.Context, addr sdk.AccAddress, amt sdk.Coins) (sdk.Coins, sdk.Tags, sdk.Error)
 	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) (sdk.Tags, sdk.Error)
+	InputOutputCoins(ctx sdk.Context, inputs []Input, outputs []Output) (sdk.Tags, sdk.Error)
```

Comment:
I don't think we want I/O coins on a sendkeeper. What was the motivation for moving these methods from `Keeper` to `SendKeeper`?

Reply:
I'm not sure why it wouldn't be here.  Like SendCoins, InputOutputCoins sends.